sources/Outputs/include/Constants.h

```diff
@@ -37,27 +37,39 @@ hash(const std::string& str) {
   return std::hash<std::string>{}(str);
 }
 
+/**
+ * @brief Computes Qmax
+ *
+ * Pow = P^2 / sqrt(P^2 + Q^2) => Q = P * sqrt(1/pow^2 - 1)
+ *
+ * @param powerFactor the power factor Pow to use
+ * @param pMax the P to use
```

Comment:
PMax

Reply:
variable convention : it must be pMax

Reply:
I mean instead of P, not instead of pMax

Reply:
OK